pkg/lib/watcher/README.md

```
@@ -70,9 +70,9 @@ db, _ := bbolt.Open("events.db", 0600, nil)
 store, _ := boltdb.NewEventStore(db)
 ```
 
-2. Create a Registry:
+2. Create a manager:
```

Markdownlint

Expected: 1; Actual: 2; Style: 1/1/1
Ordered list item prefix

(MD029, ol-prefix)